src/components/BaseSlider.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ChangeEvent, FunctionComponent, ReactElement } from "react";
+import { Typography, Slider } from "@material-ui/core";
+
+export enum Sliders {
+  contrast,
+  brightness,
+}
+
+export const sliderConfig: {
+  [id: number]: SliderConfigItem;
+} = {
+  [Sliders.contrast]: {
```

Comment:
Genuine question - is it really a base slider if it has specific configs for sliders in?

Reply:
No! I will move any config out and make it a real base slider.